src/Makefile

```diff
@@ -315,12 +318,12 @@ LIBS+=$(PNG_LDFLAGS)
 CFLAGS+=$(PNG_CFLAGS)
 endif
 
-ifdef HAVE_LIBGME
+ifndef NOGME
 OPTS+=-DHAVE_LIBGME
 
 LIBGME_PKGCONFIG?=libgme
 LIBGME_CFLAGS?=$(shell $(PKG_CONFIG) $(LIBGME_PKGCONFIG) --cflags)
-LIBGME_LDFLAGS?=$(shell $(PKG_CONFIG) $(LIBGME_PKGCONFIG) --libs)
```

Comment:
why the -lz, if you going to using zlib, add LIBGME_LDFLAGS after LIBPNG_LDFLAGS, LIBPNG_LDFLAGS have the flags flags to link to zlib